src/operations.h

```diff
@@ -98,6 +108,9 @@ struct nexttype_subparser_state {
 
         struct delegation_contents dc;  ///< wire delegation content
 
+        // Required to read Reveal public key
+        union public_key pk;  ///< wire public key
```

Comment:
This is not yet being used right?

Reply:
Yes, this field and the others above it are not used, they are only needed for `nexttype_subparser` to work properly.